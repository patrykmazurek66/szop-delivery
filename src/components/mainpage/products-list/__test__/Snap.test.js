import renderer from "react-test-renderer";
import ProductList from "../ProductsList";

it("renders correctly", () => {
  const tree = renderer.create(<ProductList categories={[]} sortType={"name-asc"} />).toJSON();
  expect(tree).toMatchSnapshot();
});
