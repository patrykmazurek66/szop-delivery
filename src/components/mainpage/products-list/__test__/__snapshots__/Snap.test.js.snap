// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="products-list"
>
  <div
    className="products-list--items"
  />
</div>
`;
